stories/components.stories.js

```diff
@@ -27,6 +29,46 @@ import website from "./assets/icon web.png";
 
 import "simplebar/dist/simplebar.min.css";
 
+const QUICK_LINKS = [
+  {
+    title: "MORE",
+    links: [
+      { href: "/about", label: "About" },
+      { href: "/faqs", label: "FAQs" },
+      { href: "/contact", label: "Contact Us" },
+    ],
+  },
+  {
+    title: "CONTACTS",
+    links: [
+      { href: "/about", label: "About" },
+      { href: "/faqs", label: "FAQs" },
+      { href: "/contact", label: "Contact Us" },
+    ],
+  },
+];
+
+storiesOf("Components/Data Display", module)
+  .addDecorator(withKnobs)
+  .add("Quick Links", () => {
+    const classes = makeStyles((theme) => ({
+      section: {
+        margin: "0 auto",
+        [theme.breakpoints.up("md")]: {
+          width: "85%",
+        },
+      },
+    }))();
+
+    return (
+      <QuickLinks
+        linkComponent={Button}
```

Comment:
Why are we using a `Button`?

Reply:
@gertie-sheshe For this,  there is no particular reason why.  I just wanted to pass the button component as `prop` to showcase it. We can also use  also`Typography` prop depending what design we follow, either outbreak or promise tracker or takwimu(not sure if it uses typography for this tho)

Reply:
Thanks @nerdlet Makes sense

Reply:
@gertie-sheshe Also maybe to mention this, the default for the list is the material ui `Link` component.
👇 
![image](https://user-images.githubusercontent.com/3693563/104436566-d0ba5c00-559e-11eb-8408-d6b8bca1a2ed.png)


So we can alternatively remove the `linkComponent` prop from QuickLinks to show something like this 👇 

![image](https://user-images.githubusercontent.com/3693563/104436591-d9129700-559e-11eb-8f38-b36c0eb1e0ff.png)